web-animations/interfaces/KeyframeEffect/target.html

```diff
@@ -253,7 +253,6 @@
   'before',
   ':abc',
   '::abc',
-  '::placeholder',
```

Comment:
Not sure why this was considered invalid

Reply:
I think they previously prohibited -webkit-prefixed pseudo-elements and maybe this is a leftover of that, but not sure.